dropwizard-hibernate/src/test/java/io/dropwizard/hibernate/SessionFactoryFactoryTest.java

```diff
@@ -50,6 +51,11 @@ public void setUp() throws Exception {
         config.setUser("sa");
         config.setDriverClass("org.hsqldb.jdbcDriver");
         config.setValidationQuery("SELECT 1 FROM INFORMATION_SCHEMA.SYSTEM_USERS");
+        
+        HashMap<String, String> props = new HashMap<>();
```

Comment:
I tend to steer away from Guava stuff through force of habit because it isn't normally worth bringing in the extra dependency just to tidy up the code a bit (IMHO) - but of course Dropwizard already uses it so there isn't really any cost.  In short, yes.